docs/scos/dev/upgradability-services/new-documentation/method-of-extended-class-overridden-on-project-level.md

```diff
@@ -0,0 +1,135 @@
+---
+title: Method of extended class is overridden one the project level
+description: Reference information for evaluator and upgrader tools.
+last_updated: Mar 23, 2022
+template: concept-topic-template
+---
+
+## Method of an extended class is overridden on the project level
+
+Factory, Dependency Provider, Repository, and Entity Manager methods belong to the private API.
```

Comment:
The best would be to have a link to a spryker doc where described private/public API
Let change the text to:
"If you extend functionality under private API, any update can cause errors or unexpected changes in functionality."